components/modal/index.en-US.md

```diff
@@ -0,0 +1,67 @@
+---
+type: Views
+category: Components
+english: Modal
+---
+
+Modal dialogs.
+
+## When to use
+
+When requiring users to interact with application, but don't want to jump to a new page to interrupt
+user's workflow, you can use `Modal` to create a new floating layer over currtent page for user 
```

Comment:
the user's workflow, you can use `Modal` to create a new floating layer over the current page for getting feedback or information purposes.